MovieSharing/ThumbnailContainer.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  Thumbnail.swift
+//  MovieSharing
+//
+//  Created by Mariam Korashi on 01.10.21.
+//
+
+import Foundation
+
+struct ThumbnailContainer: Decodable {
+    
+    // MARK: - Types
+    
+    enum CodingKeys: String, CodingKey {
+        case defaultImageData = "default"
+        case mediumImageData = "medium"
+        case highImageData = "high"
+        case standardImageData = "standard"
+    }
+    
+    // MARK: - Properties
+    
+    var defaultImageData: ImageData?
+    var mediumImageData: ImageData?
```

Comment:
- [ ] consider renaming the medium, high and standard to something like `mediumResolutionImage`